src/routes/patient.routes.test.js

```diff
@@ -0,0 +1,83 @@
+const supertest = require('supertest')
+const { app } = require('../server')
+const { db, patients } = require('../models/index')
+const request = supertest(app)
+
+beforeAll( async () => {
+  await db.sync()
+  await patients.bulkCreate([
+    {
+      id: 1,
+      first_name: 'Thor',
+      last_name: 'Odinson',
+      createdAt: Date.now(),
```

Comment:
Instead of passing in `Date.now()` we can pass in `new Date('static date')` then we can assert on the objects entirely in the remaining tests.